connection.go

```diff
@@ -173,11 +173,6 @@ func (sc *snowflakeConn) Close() (err error) {
 	glog.V(2).Infoln("Close")
 	sc.stopHeartBeat()
 
-	// ensure transaction is rollbacked
```

Comment:
all transactions would automatically rollback when the session is closed.

Reply:
hmm, I don't think the transaction is rollbacked automatically.

Reply:
Was the behavior changed?

Reply:
This is based on an conversation with Ashish. Let me add a test case to see.